### PR TITLE
Users/tashuk/sprint1902

### PR DESCRIPTION
**Pull request by TarunKrShukla**
## Description
</br>
 Brief description of the requested change.

## Checklist
- [ ] I have read the instructions mentioned in [Contribute to Code](/Contributing.md).
- [ ] I have read and understood the criteria described under [submitting changes](/Contributing.md#submitting-changes). 
- [ ] The title of the PR clearly describes the intent of the PR.
- [ ] This PR does not introduce any breaking changes to the code.

### CODE REVIEW

File: src/AzSK/Framework/Core/SVT/Services/Storage.ps1 [hunk 1]
```diff
@@ -56,7 +56,7 @@ class Storage: SVTBase
 		else{
 			$result = $result | Where-Object {$_.Tags -contains "GeneralPurposeStorage" }
 		}
-		
+		$resource = Get-AzureRmResource -ResourceId $this.ResourceContext.ResourceId;
```
Comment (VishalHaibatpure) [inline src/AzSK/Framework/Core/SVT/Services/Storage.ps1]: resource object is already present. Use $this.ResourceObject.Tags

File: src/AzSK/Framework/Core/SVT/Services/Storage.ps1 [hunk 2]
```diff
@@ -65,9 +65,14 @@ class Storage: SVTBase
 				$result = $result | Where-Object {$_.Tags -notcontains "ResourceLocked" }
 			}
 			$this.LockExists = $true;
+			$this.ControlSettings.LockedResourcesTags | ForEach-Object{
+				 if($resource.Tags.ContainsKey($_.TagName) -and $resource.Tags[$_.TagName] -eq $_.TagValue)
```
Comment (VishalHaibatpure) [inline src/AzSK/Framework/Core/SVT/Services/Storage.ps1]: Add logic to allow * for TagValue. This is required when tagName is common but value can be different for each resource

File: src/AzSK/Framework/Configurations/SVT/ControlSettings.json [hunk 3]
```diff
@@ -252,6 +252,12 @@
     "StandardGRS",
     "StandardRAGRS"
   ],
+  "LockedResourcesTags": [
+    {
+       "TagName": "application",
```
Comment (VishalHaibatpure) [inline src/AzSK/Framework/Configurations/SVT/ControlSettings.json]: can this structure be simple like "application":"databricks"

File: src/AzSK/Framework/Core/SVT/Services/Storage.ps1 (hunk 2 above)
Comment (TarunKrShukla) [reply src/AzSK/Framework/Core/SVT/Services/Storage.ps1]: i have used list of key-value pair approach to solve this issue.

File: src/AzSK/Framework/Configurations/SVT/ControlSettings.json (hunk 3 above)
Comment (TarunKrShukla) [reply src/AzSK/Framework/Configurations/SVT/ControlSettings.json]: i have used this list of tagName and tagValue based approach to handle scenario that you mentioned in second comment.

File: src/AzSK/Framework/Core/SVT/Services/Storage.ps1 (hunk 1 above)
Comment (TarunKrShukla) [reply src/AzSK/Framework/Core/SVT/Services/Storage.ps1]: Done!!!

**Pull request merged**